Makefile

```diff
@@ -24,20 +24,28 @@ deps:
 checks:
 	./scripts/checks "$(PKGS)"
 
-build: deps checks
+run-build: deps checks
 	godep go install -v $(TO_BUILD)
 
+build: start
+	vagrant ssh netplugin-node1 -c 'sudo -i bash -lc "source /etc/profile.d/envvar.sh && cd /opt/golang/src/github.com/contiv/netplugin && make run-build"'
```

Comment:
I don't see how this is any different from typical vagrant usage -- if someone does that without stopping the VM, they will be stuck with the same solution, `pkill VBoxManage` roughly.

I understand your desire but we have to figure out whether keeping the build time down or protecting the user is more important here. I don't know if there's a way to get the best of both worlds.